index.js

```diff
@@ -5,7 +5,7 @@ var _res = undefined;
 
 function load(name, doNotParse) {
   var cookies = (typeof document === 'undefined') ? _rawCookie : cookie.parse(document.cookie);
-  var cookieVal = cookies && cookies[name];
+  var cookieVal = cookies && name ? cookies[name] : cookies;
```

Comment:
Probably we should make a copy of cookies object to prevent accidental mutation. But I'm not sure.